app/src/main/java/ip/team13/petowner/data/dto/activities/ActivityDataModel.kt

```diff
@@ -0,0 +1,23 @@
+package ip.team13.petowner.data.dto.activities
```

Comment:
Can we move this model in a `data.domain` package, as it seems it is rather a model we will be using in the app, and not as a DTO.